There has been a change in the method finishHim, in the section of code starting on line nr 175.
  
The change is in a catch statement, in a try statement, in the method ```finishHim```, in the class ```TransportSearchQueryThenFetchAction```.
  
The following changes have been made:  
Changed type reference from ```java.lang.Exception``` to ```Throwable``` on line 175.  
